infrastructure/ansible/dynamic.inventory/TO.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Generate an ansible inventory compatable dataset from TO
+
+Before using, make sure you have the TrafficOps Python Native Client library available inside your env
+# Usage: python TO.py -to <TO_USER> <TO_PASS> -url to.kabletown.invalid --list
+export TO_USERNAME=<TO_USERname>
+export TO_PASSWORD=<TO_PASSword>
+export TO_URL=<TO_URL>
+"""
+
+import json
+import argparse
+import logging
+import os
+import collections
+from trafficops.tosession import TOSession
+
+# Disable rest api logging, allowing invalid certs causes warnings
+logging.getLogger('common.restapi').disabled = True
```

Comment:
This can swallow some things - are the 'warnings' you're talking about just the 'Certificate verification warnings are disabled.' warning? because nothing else should be printed.

If for some reason you REALLY can't handle having that one warning in there, consider instead raising the log level to error so that you don't swallow _everything_.

Reply:
Addressed with commit.